Sources/NIOCore/ChannelHandler.swift

```diff
@@ -346,15 +346,15 @@ extension RemovableChannelHandler {
 
 /// A `OutboundBufferedBytesAuditableChannelHandler` is a `ChannelHandler` that
 /// audits and reports the number of bytes buffered for outbound direction.
```

Comment:
Can we update this document please. Both to adjust the name of the protocol and remove the word audit